SolverInput[] solver_input_list